src/content/reference/manifest.mdx

```diff
@@ -728,16 +738,17 @@ A list of external resources that are part of your development environment.  Use
 external:
   db:
     notes: docs/database.md
+    icon: database
     endpoints:
     - name: db
       url: https://localhost:3306
   functions:
     notes: docs/lambdas.md
+    icon: function
     endpoints:
     - name: data-aggregator
       url: https://fake-id.lambda-url.us-east-1.on.aws
     - name: data-processor
-      url: https://fake-id.lambda-url.us-east-2.on.aws
```

Comment:
Do we need to remove this?

Reply:
I want to show that url is not required now that we have the dynamic endpoints enabled

Reply:
I think that removing the url value without specifying why it is not necessary is a bit confusing. Maybe we should add some kind of note, something like: it is not necessary to explicitly indicate the value of a url if it is dynamically generated in the deploy section...?

Reply:
I think it's better to add a section on dynamic endpoints, and put that there so it's more obvious.

Reply:
I added a section about how to use a dynamic endpoint. @rberrelleza can you check it?